src/core/startCoreFactory.js

```diff
@@ -50,13 +48,19 @@ function startCoreFactory(
       'core',
       coreCommand,
       [
-        '--publish=20002:20002',
+        '--service-ports',
```

Comment:
Do we need it since we already mapped it in `docker-compose.yml`?

Reply:
Yes, because `docker-compose run` ignores the ports specified in the service configuration. See docs here: https://docs.docker.com/compose/reference/run/

Reply:
Oh.. I see. Good to know, thank you.